app/src/main/java/org/fossasia/openevent/app/events/EventsListAdapter.java

```diff
@@ -5,69 +5,41 @@
 import android.support.v4.app.FragmentManager;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.RecyclerView;
-import android.text.TextUtils;
 import android.view.LayoutInflater;
-import android.view.View;
 import android.view.ViewGroup;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import com.squareup.picasso.Picasso;
 
 import org.fossasia.openevent.app.R;
 import org.fossasia.openevent.app.data.models.Event;
+import org.fossasia.openevent.app.databinding.EventLayoutBinding;
 import org.fossasia.openevent.app.event.EventContainerFragment;
 
 import java.util.List;
 
-import butterknife.BindView;
-import butterknife.ButterKnife;
-
-
 class EventsListAdapter extends RecyclerView.Adapter<EventsListAdapter.EventRecyclerViewHolder>{
 
     private List<Event> events;
     private Context context;
 
     private boolean isTwoPane;
 
-    public EventsListAdapter(List<Event> events, Context context, boolean isTwoPane) {
+    EventsListAdapter(List<Event> events, Context context, boolean isTwoPane) {
         this.events = events;
         this.context = context;
         this.isTwoPane = isTwoPane;
     }
 
     @Override
     public EventRecyclerViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
-        View itemView = LayoutInflater.from(parent.getContext()).inflate(R.layout.event_layout, parent, false);
+        LayoutInflater layoutInflater = LayoutInflater.from(parent.getContext());
 
-        return new EventRecyclerViewHolder(itemView);
+        EventLayoutBinding binding = EventLayoutBinding.inflate(layoutInflater, parent, false);
+        return new EventRecyclerViewHolder(binding);
     }
 
     @Override
     public void onBindViewHolder(final EventRecyclerViewHolder holder, int position) {
-
         final Event thisEvent = events.get(position);
-        holder.eventTitle.setText(thisEvent.getName());
-
-        String thumbnail = thisEvent.getThumbnail();
-
-        if(thumbnail != null && !TextUtils.isEmpty(thumbnail)) {
```

Comment:
@iamareebjamal This checking is missed after the PR. Now app crashes in `EventsListActivity` on empty thumbnail field. Please make required changes.